Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -536,6 +514,42 @@ defineSuite([
         });
     });
 
+    it('replacement refinement - refines to visible ready children', function() {
```

Comment:
With all the tileset traversal changes, are you sure this is the only new test we need?  Are all cases covered?